sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -474,6 +479,54 @@ case class Lag(input: Expression, offset: Expression, default: Expression)
   override val direction = Descending
 }
 
+/**
+ * The NthValue function returns the value of `input` at the `offset`th row of beginning
```

Comment:
`of beginning` -> `from beginning of the window partition.`

Reply:
OK